src/view/uniprotkb/components/SubcellularLocationView.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { FC, Fragment } from 'react';
+import { SubcellularLocationComment } from '../../../model/types/CommentTypes';
+import UniProtEvidenceTag from '../../../components/UniProtEvidenceTag';
+import { TextView } from './FreeTextView';
+
+const SubcellularLocationView: FC<{
+  comments: SubcellularLocationComment[];
+}> = ({ comments }) => {
+  return (
+    <Fragment>
+      {comments.map(
+        subcellData =>
+          subcellData.subcellularLocations && (
+            <section
+              className="text-block"
+              key={subcellData.molecule ? subcellData.molecule : 'undef'}
```

Comment:
Having `'undef'` as a fallback key could be problematic. A uuid would be a little safer.

Reply:
Added v1 when no molecule